root/etc/systemd/system/enable-promiscuous-mode@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Control promiscuous mode for interface %i
+After=network-online.target
+Wants=network-online.target
+ConditionKernelCommandLine=!fdi.promisc
```

Comment:
Works great, can you change this to `fdi.nopromis` tho? It's actually flag that disables it. Make no changes as this is already tested! :-)

Reply:
@lzap done - force-pushed my branch again with adjustment.